samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs

```diff
@@ -102,4 +109,34 @@ public SemanticKernelController(IServiceProvider serviceProvider, IConfiguration
 
         return this.Ok(new AskResult { Value = result.Result, Variables = result.Variables.Select(v => new KeyValuePair<string, string>(v.Key, v.Value)) });
     }
+
+    private async Task RegisterOpenApiSkills(PluginAuthHeaders pluginAuthHeaders, Kernel kernel)
+    {
+        if (pluginAuthHeaders.GithubAuthentication != null)
+        {
+            var authenticationProvider = new BearerAuthenticationProvider(() => { return Task.FromResult(pluginAuthHeaders.GithubAuthentication); });
+            this._logger.LogInformation("Registering GitHub Skill, token {0}", pluginAuthHeaders.GithubAuthentication);
+
+            var skill = await kernel.ImportOpenApiSkillFromResourceAsync(SkillResourceNames.GitHub,
+             authenticationProvider.AuthenticateRequestAsync);
+        }
+
+        // if (pluginAuthHeaders.GraphAuthentication != null)
```

Comment:
I appreciate the guidance, but let's keep these dead code sections out of the codebase until we bring in each plugin.

Reply:
Removed